packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -30,7 +33,102 @@ class Console extends StatefulWidget {
 }
 
 class _ConsoleState extends State<Console> {
-  final scrollController = ScrollController();
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<List<String>>(
+      valueListenable: widget.controller.stdio,
+      builder: (context, lines, _) {
+        // Extract this to a reusable ConsoleWithControls StatelessWidget?
```

Comment:
I'd remove this comment, or convert it into a TODO(username): comment.

Reply:
Yep, will do.

Reply:
In fact, I'm going to do what the comment suggests so I can get rid of it :P